src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/InvokeRestMethodCommand.Common.cs

```diff
@@ -85,7 +85,7 @@ public int MaximumFollowRelLink
 
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
Validate Update-Help from the Web for all PowerShell modules..Validate Update-Help for module 'CimCmdlets' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```